### ISSUE TITLE
Update Docs to be more helpful 

### ISSUE THREAD

**Issue by murffious** [Azure-Samples/storage-blob-upload-from-webapp-node#2]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>
Follow Steps EXACTLY as mentioned in the READ me leads to connection error 
### Any log messages given by the failure
>
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
### Expected/desired behavior
>
actually work and connect and load html to use
### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>
NOTHING here is relevant 
### Mention any other details that might be useful
All you have to do is simply help users know when they create a container online in Azure to grab the name of the container and update the line 10 in routes/index.js with your container name.  As a FIRST time user it took me almost an hour to figure this out .  NOT COOL - I have to name my container 'thumbnails' to make the code work as is. BUT that is not in the docs. 
> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by Jozott00** [comment]
You are right! It is not that difficult to write in the docs that the container has to be renamed to "Thumbnails"!
